index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<style>
+    img {
+        display: block;
+        max_width: 100%;
+
+
+    }
+
+    #image {
+        max-width: 100%;
+        width: 100%;
+    }
+</style>
+
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p> The man who saved a billion lives</p>
+        <div id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+                alt="image">
+            <figcaption id="img-caption"> Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to
+                increase wheat yields - part of his life-long war on hunger. </figcaption>
+            </img>
+            <div id="tribute-info">
+                <h3> Here's a time line of Dr. Borlaug's life:</h3>
+                <ul>
+
+
```

Comment:
I suggest you remove the empty blank spaces in your code as it may expose your code 😊